pkg/diff/diff.go

```diff
@@ -188,14 +185,14 @@ func (t *TableDiff) getTableInfo(ctx context.Context) error {
 	if err != nil {
 		return errors.Trace(err)
 	}
-	t.TargetTable.info = removeColumns(tableInfo, t.RemoveColumns)
+	t.TargetTable.info = removeColumns(tableInfo, t.IgnoreColumns)
```

Comment:
if this column is index column, I think we should remove this index all

Reply:
we use tidb's statistical information which is treat index as a unit to split chunk, so if removed any column in an index, this index's  statistical information is unavailable.